echopype/tests/convert/test_parsed_to_zarr.py

```diff
@@ -68,44 +91,70 @@ def test_raw2zarr(raw_file, sonar_model, offload_to_zarr, ek60_path):
         assert os.path.exists(output_save_path)
 
 
-@pytest.mark.skip(reason="Full testing of writing variables directly to a zarr store has not been implemented yet.")
-def test_writing_directly_to_zarr(ek60_path, ek80_path):
+@pytest.mark.parametrize(
+    ["path_model", "raw_file", "sonar_model"],
+    [
+        ("EK60", "ncei-wcsd/Summer2017-D20170615-T190214.raw", "EK60"),
+        ("EK60", "DY1002_EK60-D20100318-T023008_rep_freq.raw", "EK60"),
+        ("EK80",  "Summer2018--D20180905-T033113.raw", "EK80"),
+        ("EK80_CAL", "2018115-D20181213-T094600.raw", "EK80"),
+        ("EK80",  "Green2.Survey2.FM.short.slow.-D20191004-T211557.raw", "EK80"),
+        ("EK80",  "2019118 group2survey-D20191214-T081342.raw", "EK80"),
```

Comment:
@b-reyes : I couldn't find what type of file is `2018115-D20181213-T094600.raw`, do you know what the particularity is ?

Two other files that would be good to add are:
- `raw4-D20220514-T172704.raw`
- `D20210330-T123857.raw`

Reply:
@leewujung I think `2018115-D20181213-T094600.raw` is a good test case because it is an EK80 file with only 1 beam group corresponding to complex samples. Thus, there is no `power` and `angle` data. Beyond this characteristic, I am not sure what other particularities there are.

@leewujung thank you, I will add `raw4-D20220514-T172704.raw`.